src/styles/blocks/card.scss

```diff
@@ -0,0 +1,104 @@
+$blueAccent: #00acdc;
+$mainAccent: #060b35;
+$secondary: #616070;
+$grayElements: #f3f3f3;
+
+@mixin flex-with-space-between {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid $grayElements;
+  border-radius: 5px;
+
+  &__image {
+    width: 160px;
+  }
+
+  &__title {
+    color: $mainAccent;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+
+    // не впевнений що margin можна додавати
```

Comment:
Of course you can cause this is the BEM element, so here you can, but not in BEM block